gen/testdata/golden.go

```diff
@@ -8,6 +8,15 @@ import (
 	"github.com/hamba/avro/v2"
 )
 
+type Cards string
+
+const (
+	SPADES   Cards = "SPADES"
```

Comment:
This should not be all upper. It should also be prefixed with the type. So `CardSpades`.